### DISCUSSION TITLE
[Question] Example Code Not Running Properly

### DISCUSSION THREAD

**Discussion by Wharvey55** [theOehrly/Fast-F1#415] [Q&A]
Hello all! I was trying to run the example code for the driver laptime distribution visualization. However when I ran it in a Conda Spyder IDE environment, it did not generate the graph properly, and was missing the dots for the laptimes. It spit out the error:

AttributeError: module 'fastf1.plotting' has no attribute 'COMPOUND_COLORS'

I also ran it in a Jupyter Notebook and got the same problem there too. 

![download](https://github.com/theOehrly/Fast-F1/assets/139138195/68af339f-7ed6-4d7d-924b-c0f68c04ca1d)

Not sure what I am missing that caused it to give me the error since I just downloaded the code and was trying to run it, any help would be appreciated!

**Comment by Casper-Guo** [comment]
The `COMPOUND_COLORS` attribute only came out in version 3.0.0 so you should first check whether you are running the latest version of FastF1

**Reply by Wharvey55** [reply]
Thanks for the helpful tip! I'll be sure to try that a little later today.
